src/components/Kv/KvLightbox.vue

```diff
@@ -156,7 +159,7 @@ export default {
 
 			.lightbox-columns {
 				position: relative;
-				width: 100%;
```

Comment:
This allows the lightbox to be smaller than 100% if the content is narrow.

Reply:
We might want to double-check if anything was relying on this for centering the content within the lightbox

Reply:
Open all the lightboxes! I'll take a look through this today :)

Reply:
Going to break lightbox changes into a different PR.